### PR TITLE
Switch to `reticulate` focused installation instructions

### LINKED ISSUE #159: Encourage use of reticulate's installation process of python package
One way to get around some of the installation challenges and associated path issues may be to use reticulate's installation facilitities:

https://rstudio.github.io/reticulate/articles/python_packages.html

This could either be in the installation guidelines or we might even be able to trigger the installation on package load if R doesn't see the package.

### PR DESCRIPTION
**Pull request by ethanwhite**
Fixes #159

### CODE REVIEW

**Comment by ethanwhite** [comment]
If there are other places this should go as well let me know. Probably on the website. Maybe somewhere in the docs?

**Comment by ethanwhite** [comment]
Just rebased on `master` after the testing updates (so hopefully this will pass now).

File: README.md [hunk 1]
```diff
@@ -17,29 +17,61 @@ and more time analyzing it.
...
+* Use this if you are already familiar with Python and have a local Python installation*
+
+1. Check that your local Python installation is Python 3
+2. In R install the `reticulate` package:
+
+  ```coffee
+  install.packages("reticulate")
+  ```
+
+3. In R run the following (*replacing "/path/to/python" with the path to you Python executeable*) to install the `retriever` Python package:
```
Comment (henrykironde) [inline README.md]: Could we change from * to use `, It renders more clearly.

**Comment by ethanwhite** [comment]
Thanks for the review @henrykironde. I just updated the PR with those two changes.

**Comment by henrykironde** [comment]
Thanks Ethan.

**Pull request merged**